packages/react-core/src/components/Dropdown/InternalDropdownItem.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import classNames from 'classnames';
```

Comment:
Can you not use the import below this line (`import { css } from '@patternfly/react-styles';`)? `classnames` technically isn't a dependency of react-core and this would be the first time we depend on it in react-core.

Reply:
Fixed. I think I subconsciously thought I was doing product work.